tests/framework/e2e/downgrade.go

```diff
@@ -117,11 +161,16 @@ func DowngradeUpgradeMembersByID(t *testing.T, lg *zap.Logger, clus *EtcdProcess
 			return err
 		}
 	}
+
+	clusterVersion := targetVersion.String()
+	if !isDowngrade && len(membersToChange) != len(clus.Procs) {
+		clusterVersion = currentVersion.String()
+	}
```

Comment:
Can we revert this change as well to make this PR as simple as possible? It unnecessarily complicates the test case. 

When upgrading, the current test case ensures that all members will be on the target version.

```suggestion
```

Reply:
But it's bug in this function....

The robustness test failure is caused by this.
If we remove this, the robustness test always fails when the failpoint is related to downgrade.

Reply:
> When upgrading, the current test case ensures that all members will be on the target version.

This isn't true in robustness test?